docs/en_US/Compression/Tutorial.rst

```diff
@@ -0,0 +1,190 @@
+
+
+* **quant_bits** : int or dict of {str : int}
+
+bits length of quantization, key is the quantization type, value is the quantization bits length, eg. 
+
+.. code-block:: bash
+
+   {
+       quant_bits: {
+           'weight': 8,
+           'output': 4,
+           },
+   }
+
+when the value is int type, all quantization types share same bits length. eg. 
+
+.. code-block:: bash
+
+   {
+       quant_bits: 8, # weight or output quantization are all 8 bits
+   }
+
+The following example shows a more complete ``config_list``\ , it uses ``op_names`` (or ``op_types``\ ) to specify the target layers along with the quantization bits for those layers.
+
+.. code-block:: bash
+
+   config_list = [{
+           'quant_types': ['weight'],        
+           'quant_bits': 8, 
+           'op_names': ['conv1']
+       }, {
+           'quant_types': ['weight'],
+           'quant_bits': 4,
+           'quant_start_step': 0,
+           'op_names': ['conv2']
+       }, {
+           'quant_types': ['weight'],
+           'quant_bits': 3,
+           'op_names': ['fc1']
+           },
+          {
+           'quant_types': ['weight'],
+           'quant_bits': 2,
+           'op_names': ['fc2']
+           }
+   ]
+
+In this example, 'op_names' is the name of layer and four layers will be quantized to different quant_bits.
+
+
+Export compression result
+-------------------------
+
+Export the pruend model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can easily export the pruned model using the following API if you are pruning your model, ``state_dict`` of the sparse model weights will be stored in ``model.pth``\ , which can be loaded by ``torch.load('model.pth')``. Note that, the exported ``model.pth``\ has the same parameters as the original model except the masked weights are zero. ``mask_dict`` stores the binary value that produced by the pruning algorithm, which can be further used to speed up the model.
+
+.. code-block:: python
+
+   # export model weights and mask
+   pruner.export_model(model_path='model.pth', mask_path='mask.pth')
+
+   # apply mask to model
+   from nni.compression.pytorch import apply_compression_results
+
+   apply_compression_results(model, mask_file, device)
+
+
+export model in ``onnx`` format(\ ``input_shape`` need to be specified):
+
+.. code-block:: python
+
+   pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
+
+
+Export the quantized model
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can export the quantized model directly by using ``torch.save`` api and the quantized model can be loaded by ``torch.load`` without any extra modification. The following example shows the normal procedure of saving, loading quantized model and get related parameters in QAT.
+
+.. code-block:: python
+   
+   # Save quantized model which is generated by using NNI QAT algorithm
+   torch.save(model.state_dict(), "quantized_model.pkt")
```

Comment:
just curious, what is `.pkt` mean, why not use `.pth`?

Reply:
@linbinskn

Reply:
`.pkt` means 'Packet Tracer Network Simulation Model'. The reason why I use '.pkt' here is that I think model generated from QAT is simulated model not the real model. Of course we can use `.pth` here.

Reply:
`.pth` is be better

Reply:
fix it